rust/tests/behaviour/main.rs

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+use cucumber::{gherkin::Step, given, then, when, World};
+
+use typeql_lang::parse_query;
+
+#[derive(Debug, Default, World)]
+pub struct TypeQLWorld;
+
+fn main() {
+    futures::executor::block_on(
+        TypeQLWorld::cucumber().filter_run("../vaticle_typedb_behaviour", |_, _, sc| {
```

Comment:
Where does this path come from - and how reliable is it? i.e. would the path still be `../vaticle_typedb_behaviour` if we refactored this package and moved it somewhere else?

It would be beneficial to future readers to clarify in the code - either here or in the `BUILD` file - what this directory is and how it comes about.